jdbc/src/main/java/site/ycsb/db/flavors/CockroachDBFlavor.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2016 YCSB contributors. All rights reserved.
```

Comment:
is this year correct?

Reply:
will copy/paste from [CONTRIBUTING.md](https://github.com/brianfrankcooper/YCSB/blob/master/CONTRIBUTING.md) latest from which has `2015-2017`